### PR TITLE
doc: build

### PR DESCRIPTION
**Pull request by xiang90**
/cc @heyitsanthony @gyuho @joshix

### CODE REVIEW

File: Documentation/dl_build.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Download and Build
+
+## System Requirement
```
Comment (heyitsanthony) [inline Documentation/dl_build.md]: s/Requirement/Requirements/

File: Documentation/dl_build.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+$ git clone github.com:coreos/etcd.git
+$ cd etcd
+$ ./build
+$ ./bin/etcd
+...
+```
+
+## Test your Installation
+
+Check etcd binary is built correctly by starting etcd and setting a key.
```
Comment (heyitsanthony) [inline Documentation/dl_build.md]: s/Check etcd/check the etcd/

**Comment by heyitsanthony** [comment]
lgtm aside from minor nits

**Comment by gyuho** [comment]
LGTM.

File: Documentation/dl_build.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# Download and Build
+
+## System Requirements
+
+TODO
+
+## Download the Pre-built Binary
+
+The easiest way to get etcd is to use one of the pre-built release binaries which are available for OSX, Linux, Windows, AppC (ACI), and Docker. Instructions for using these binaries are on the [GitHub releases page][github-release].
```
Comment (joshix) [inline Documentation/dl_build.md]: appc

**Comment by joshix** [comment]
"appc" is preferred. Otherwise LGTM.

**Pull request merged**